redi-connect-front/src/components/templates/Account.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import Footer from "../organisms/Footer";
+import Logo from "../atoms/Logo";
+
+import { Container, Section } from "react-bulma-components";
+
+interface Props {
+  children: React.ReactNode;
+}
+
+const Account = ({ children }: Props) => (
```

Comment:
I was searching to differentiate between user `account` and user `profile`. So I decided to go here with account for the layout all around the login, reset, signup and the profile will be another type. `FormWrapper` would meant that I would use this on every page where there is form which would not be correct. 
Let'S move on with this naming for now, if something better comes up I will change. It's still early to have a fixed naming on some things. Layout will change

Reply:
I'm starting to see the atomic design patterns here :) Did you bookmark any favourite "atomic design as applied to react projects" articles you can share?

About naming for `Account` ... how about `AccountOperation`? Account by itself still seems way too generic.

Reply:
Well adding Operation doesn't add any value for me. Everything will work the same and mean the same. So I guess I just update and move on.

Reply:
Btw, I dont know if there is an article about how to apply atomic design to react. I don't know if it would be much different that the original approach. I still think that https://bradfrost.com/blog/post/atomic-web-design/ gives a good overview. Some things can be differently understood so in every team there should be talks about the ways that are applied.

Reply:
Ugh, I still don't like just "Account". I'll sleep on it for now until I think of a beautiful diamond term that perfectly encapsulates everything 😇🤓

I looked around for atomic + react, and found this one: https://cheesecakelabs.com/blog/rethinking-atomic-design-react-projects/

It pretty much seems to be exactly what we're doing. It's useful to read to really let it sink in. Here's one highlight: "Pages are not components, as these are just instances of Templates, and the pages should be closer to the service layer of your application".

Can we link to that article from `README.md`? As a "reference resource"?